modules/ols-0-3-5-release-notes.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-3-5-release-notes_{context}"]
+= {ols-long} version 0.3.5
+
+{ols-official} 0.3.5 is now available on {ocp-product-title} 4.15 and later.
+
+[IMPORTANT]
+====
+{ols-official} is designed for Federal Information Processing Standards (FIPS). When running on {ocp-product-title} in FIPS mode, it uses the {rhel} cryptographic libraries submitted (or planned to be submitted) to NIST for FIPS validation on only the `x86_64`, `ppc64le`, and `s390X` architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status of the individual versions of {rhel} cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].
```

Comment:
FYI, the second link opens a page entitled "Product compliance".

Reply:
Thank you here, too, @lahinson! This changed in the broader org at some point and I had no idea. I appreciate your attention to detail.